src/types/views.ts

```diff
@@ -43,4 +43,6 @@ export type OverKeyboardViewProps = PropsWithChildren<{
    * A boolean prop indicating whether the view is visible or not. If it's true then view is shown on the screen. If it's false then view is hidden.
    */
   visible: boolean;
+  height?: number;
+  width?: number;
```

Comment:
I'd avoid passing style-props as separate props 🤔 

Maybe we should add optional `style` property? It would be a little bit more universal 👀